ask-sdk-core/src/com/amazon/ask/response/ResponseBuilder.java

```diff
@@ -85,6 +85,17 @@ public ResponseBuilder withSpeech(String speechText) {
         return this;
     }
 
+    /**
+     * Sets a Card {@link Card} object on the response.
```

Comment:
Don't need to have 'Card' as plaintext and a link. Just keep the link
```suggestion
     * Sets a {@link Card} object on the response.
```

Reply:
Fixed it.